src/sas/cli.py

```diff
@@ -180,6 +177,7 @@ def main(logging="production"):
             exitmsg = banner = ""
         else:
             import platform
+
```

Comment:
Same complaint about imports not on top.
In this (and maybe other) case, it might be useful to have delayed imports for performance reasons, but then we should add  `# isort: skip` or `# noqa: I001` for the offending line(s)

Reply:
In this case, the imports are not module-level imports, they are contained within the routine `main`. Therefore, if we want these as module-level imports, the module must be manually refactored.

Reply:
I've seen that there is a ruff rule for this, albeit one I have never used before and which does not have automatic fixes available. The rule is [PLC0415](https://docs.astral.sh/ruff/rules/import-outside-top-level/), and there are 179 violations of this rule within SasView. Given there are no automatic fixes, we remain in the position that changing this requires manually refactoring the code, so I do not recommend including it in our ruleset.